positioning/styles.css

```diff
@@ -0,0 +1,139 @@
+/*using custom properties, we're defining where the z-index actually "is" */
+:root {
+  --z-1: 100;
+  --z-2: 200;
+  --z-3: 300;
+  --z-4: 400;
+  --z-5: 500;
+  --z-6: 600;
+  --z-7: 700;
+  --z-8: 800;
+  --z-9: 900;
+}
+
+body {
+  margin: 0;
+  font: 1rem / 1.5 sans-serif;
+}
+
+main {
+  /* border: 1px solid red; */
+  margin: 6rem auto;
+  width: 95%;
+  max-width: 75ch;
+  font-size: 1.125rem;
+}
+
+figure {
+  margin: 2rem 0;
+  padding: 0;
+  position: relative;
+  z-index: var(--z-1);
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+h2 {
+  font: 400 1.875rem/1.25 MuseoModerno, sans-serif;
+  color: hsl(300, 40%, 30%);
+}
+
+figcaption {
+  /* position: relative; */
+  /* top: -4rem; */
+  position: absolute;
+  right: 0;
+  bottom: 1rem;
+  display: inline-block;
+  color: white;
+  background-color: hsl(210, 40%, 10%);
+  padding: 0.25rem 1rem;
+}
+
+figcaption a {
+  color: inherit;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: var(--z-9);
```

Comment:
It's usually best practice to keep your z-index in order incrementally since the lowest defined z-index on the page is here was 1, a z-index of 2 wouldve been fine here. But considering the context here that this is a fixed header and you'll always want to it to be stacked above the rest of the page this a fine premature optimization.

Reply:
Ok I'm sort of answering both of your comments here. Like you said, there's some weird context in the tutorial. I think some of the reasons we didn't follow the z-index more sequentially (or like you said, not even bother defining it in some cases), is literally to show concepts in the tutorial.